File: cpp/bc1099.cpp
#include <iostream>

using namespace std;

int main()
{
	int num, fVal, sVal;
	
	cin >> num;
	
	for(int i = 0; i < num; i++)
	{
		int sOdd = 0;
		
		cin >> fVal >> sVal;
		
		if(sVal < fVal)
		{
			int aux = fVal;
			fVal = sVal;
			sVal = aux;
		}
		
		for(int j = fVal+1; j < sVal; j++)
			if(j % 2 != 0) sOdd += j;
			
		cout << sOdd << endl;
	}
}

File: python/bc1099.py
num = int(input())
bigger = 0
smaller = 0

for c in range(0, num):
    sumI = 0
    pVal = input().split()

    fVal = int(pVal[0])
    sVal = int(pVal[1])

    if fVal > sVal:
        bigger = fVal
        smaller = sVal
    else:
        bigger = sVal
        smaller = fVal

    for d in range(smaller+1, bigger):
        if d % 2 != 0: sumI += d

    print(sumI)